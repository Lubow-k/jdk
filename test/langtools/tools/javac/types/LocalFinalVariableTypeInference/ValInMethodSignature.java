/*
 * @test
 * @compile/fail/ref=ValInMethodSignature.out -XDrawDiagnostics ValInMethodSignature.java
 */


class ValInMethodSignature {
    // 'val' in method signature
    // error
    void test(val number) { }
}
