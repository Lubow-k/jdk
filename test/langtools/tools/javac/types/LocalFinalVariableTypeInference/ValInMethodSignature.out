ValInMethodSignature.java:10:15: compiler.err.restricted.type.not.allowed.here: val
1 error